src/librustdoc/html/static/js/main.js

```diff
@@ -1053,15 +1060,18 @@ function preLoadCss(cssUrl) {
         innerToggle.children[0].innerText = "Show all";
     }
 
-    function toggleAllDocs() {
+    /**
+     * @param {MouseEvent=} ev
```

Comment:
Shouldn't it be:

```suggestion
     * @param {MouseEvent} ev
```

?